modules/oadp-mcg-performance-tuning.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-mcg.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-mcg-performance-tuning_{context}"]
+= Performance tuning guide for Multicloud Object Gateway
+
+The default configuration of {rh-storage} for Multicloud Object Gateway is optimized for low resource consumption and not performance.
+
+
+* Large files: In the case of large files, the metadata-to-data ratio is low. Increasing the resources for the endpoints (memory and CPU) and the number of endpoints would be the first thing to do. In the case of namespace buckets, only memory would be sufficient. 
++
+The CPU is important mainly for data buckets, where the endpoints use the CPU for encryption and deduplication.
+
+* * Small objects: In the case of small objects, the metadata-to-data ratio is high. For data buckets, this means a high level of resource involvement from the core and database. Increasing those pods resources would be the first step.
++
+Endpoints memory will probably not be under pressure if the core and database respond quickly. If they do not respond quickly enough, then pressure will build, and endpoints will eventually be under pressure. In this case, you should increase both core and database, with more emphasis on the database itself.
+
+* High amount of configuration entities such as large number of buckets, accounts: This would also point to the DB and core, with more emphasis on the core.
```

Comment:
```suggestion
* A high number of configuration entities, such as a large number of buckets or accounts: This would also point to the database and core, with more emphasis on the core than the database.
```